node/src/actors/chain_manager/mining.rs

```diff
@@ -125,6 +128,25 @@ impl ChainManager {
 
         let own_pkh = self.own_pkh.unwrap_or_default();
 
+        let superblock_period = u32::from(chain_info.consensus_constants.superblock_period);
+        if rep_engine.ars().contains(&own_pkh) && current_epoch % superblock_period == 0 {
+            // FIXME(#1236): ARS Members have to use the BLS address when forwarding
+            let ars_members: Vec<PublicKeyHash> = rep_engine
```

Comment:
Shouldn't this be the ARS members from 2 epochs ago?

Reply:
It will be modified after Reputation Merkelitation improvement

Reply:
I added a TODO